File: leetcode/Java Solutions/RunningSum.java
/**
 * Given an array nums. We define a running sum of an array as runningSum[i] =
 * sum(nums[0]…nums[i]). Return the running sum of nums.
 */
public class RunningSum {
    public int[] runningSum(int[] nums) {
        int rsum = nums[0];
        for (int i = 1; i < nums.length; i++) {
            rsum += nums[i];
            nums[i] = rsum;
        }

        return nums;
    }
}

File: leetcode/Javascript Solutions/runningSum.js
ram {number[]} nums
 * @return {number[]}
 *  */
	let runningSum = (nums)=>{
		    let sum = nums[0]
		    for(let i=1;i<nums.length;i++){
			            sum+=nums[i]
			            nums[i]=sum
			        }
		    
		    return nums
	}
